RR.App/Controllers/HRIS/EmployeeDocumentController.cs

```diff
@@ -18,13 +18,13 @@ public EmployeeDocumentController(IEmployeeDocumentService employeeDocumentServi
     }
 
     [Authorize(Policy = "AllRolesPolicy")]
-    [HttpGet("{employeeId}")]
-    public async Task<IActionResult> GetAllEmployeeDocuments(int employeeId)
+    [HttpGet("all/{employeeId}/{documentType}")]
+    public async Task<IActionResult> GetAllEmployeeDocuments(int employeeId, int documentType)
     {
         try
         {
-            var employeeDocuments = await _employeeDocumentService.GetAllEmployeeDocuments(employeeId);
-            return Ok(employeeDocuments);
+                var employeeDocuments = await _employeeDocumentService.GetAllEmployeeDocuments(employeeId, (DocumentType)documentType);
```

Comment:
You can just say GetEmployeeDocuments and the parameters show that it's by employeeId and documentType